js/options.js

```diff
@@ -9,20 +9,21 @@ $(function() {
         }
     });
     $('#save').on('click', function() {
-        getTenants(function(tenants) {
-            for (var tenantCode in tenants) {
-                if (tenantCode === $('#tenantCode').val()) {
-                  saveActiveTenant(tenants[tenantCode], function() {
```

Comment:
extracted this into a function `saveActiveTenantAndUpdateStatus` now that I need to do it in two places